src/data/dataCardContent.ts

```diff
@@ -0,0 +1,34 @@
+import { CardContentProps } from "@/types/dataCardContent";
+
+const dataCardContent: CardContentProps[] = [
+  {
+    id: "1",
+    initialSection: {
+      title: "Fio dental para crianças:",
+      mainText: "O fio dental é tão importante quanto a escovação, mas muitas vezes é esquecido, especialmente quando se trata de crianças. Vamos descobrir como e quando introduzir o fio dental para os pequenos.",
+      img: "/assets/fiodental-3.png",
+      subtitle: "Quando começar:",
+      secondText: "Assim que dois dentes estiverem em contato, é hora de começar a usar o fio dental. Isso geralmente acontece por volta dos 2-3 anos."
```

Comment:
No Figma, o texto "2-3 anos" é em negrito. Pra conseguir esse efeito quando o conteúdo vem de um arquivo externo, a gente precisar elaborar alguma estratégia. Acho melhor deixar assim por enquanto e conversar com o time se textos podem vir com efeitos como negrito e itálico e, caso eles confirmem, aí a gente pensa em como aplicar esse efeito.

Caso o pessoal de QA reporte como bug, aí acho mais fácil colocar o texto manualmente de maneira temporária.